capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -121,6 +132,15 @@ class ClubPage extends Component {
         </Row>
         <div className="description"> {this.state.club.description} </div>
         {bookTile}
+        {isOwner && 
+          <SearchBookModal
+            objectId={this.props.id}
+            update={this.handleBookChange}
+            text="Change the Club's Book"
+            putURL="/api/clubs"
```

Comment:
nit: Use single quotes for string literals (https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes).

Reply:
I'll change this!